### PR TITLE
OCD-3656 : AQA test for non prod environment indicator

### PR DESCRIPTION
**Pull request by palakpatel311**

### CODE REVIEW

File: e2e/components/environment-indicator/environment-indicator.po.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class EnvironmentIndicatorComponent {
+  constructor() {
+    this.elements = {
+      nonprod: '#non-prod-indicator',
+    };
+  }
+
+  get nonProdIndicator() {
+    return $(this.elements.nonprod);
```
Comment (andlar) [inline e2e/components/environment-indicator/environment-indicator.po.js]: If the "get" function says `nonProdIndicator` with the camelCase name, should the element also be camel case? `elements.nonProd`?

**Review by ashwinimore — DISMISSED** [review]

**Review by andlar — DISMISSED** [review]

**Review by athomas-git — DISMISSED** [review]

**Review by ashwinimore — DISMISSED** [review]

File: e2e/pages/organizations/onc-acbs-atls/acbs.readonly.spec.js [hunk 2]
```diff
@@ -21,8 +21,8 @@ describe('the ONC-ACB Management page', () => {
   };
 
   beforeEach(async () => {
-    browser.setWindowSize(1600, 1024); // demo of a bigger screen (esp. useful for screenshots)
-    browser.setWindowRect(0, 0, 1600, 1024); // not sure if both are required
+    browser.setWindowSize(1600, 1024);
+    browser.setWindowRect(0, 0, 1600, 1024);
```
Comment (andlar) [inline e2e/pages/organizations/onc-acbs-atls/acbs.readonly.spec.js]: Are both of these required?

**Review by ashwinimore — APPROVED** [review]

File: e2e/pages/organizations/onc-acbs-atls/acbs.readonly.spec.js [hunk 3]
```diff
@@ -21,8 +21,6 @@ describe('the ONC-ACB Management page', () => {
   };
 
   beforeEach(async () => {
-    browser.setWindowSize(1600, 1024); // demo of a bigger screen (esp. useful for screenshots)
-    browser.setWindowRect(0, 0, 1600, 1024); // not sure if both are required
```
Comment (andlar) [inline e2e/pages/organizations/onc-acbs-atls/acbs.readonly.spec.js:25]: Why are these resize things being removed at all? Are they not useful?

Comment (palakpatel311) [reply e2e/pages/organizations/onc-acbs-atls/acbs.readonly.spec.js:25]: Window resizing is done in hooks method itself while opening URL so not needed here.

**Review by andlar — APPROVED** [review]

**Review by athomas-git — APPROVED** [review]

**Pull request merged**